ceph.spec.in

```diff
@@ -134,6 +134,9 @@ BuildRequires:	snappy-devel
 BuildRequires:	udev
 BuildRequires:	util-linux
 BuildRequires:	valgrind-devel
+%if %{with make_check}
+BuildRequires:	which
+%endif
```

Comment:
Yeah, I looked at the sources and it would not be as easy as I have hoped for.

btw: I would be ok if there was no condition around it.